pages/agent/v3/unclustered_tokens.md

```diff
@@ -0,0 +1,143 @@
+# Unclustered agent tokens
+
+> 🚧 This page documents a deprecated feature of Buildkite
+> Previously, agents only connected to Buildkite directly via a token which was created and managed by the processes described on this page. These tokens are now a deprecated feature of Buildkite, and are referred to as _unclustered agent tokens_. Unclustered agent tokens, however, are still available to customers who have not yet migrated their pipelines across to a [cluster](/docs/clusters/overview).
```

Comment:
```suggestion
> Previously, agents only connected to Buildkite via a token which was created and managed by the processes described on this page. These tokens are now a deprecated feature of Buildkite, and are referred to as _unclustered agent tokens_. Unclustered agent tokens, however, are still available to customers who have not yet migrated their pipelines to a [cluster](/docs/clusters/overview).
```

Reply:
Yeah, I wasn't keen on the placement of the word 'directly' here but here, I'm using this word to contrast how unclustered agent tokens connect to Buildkite with what we now call 'agent tokens' (formerly 'cluster tokens').

The newer 'agent tokens' connect to Buildkite via a cluster, whereas 'unclustered agent tokens' connect directly to Buildkite.

Therefore, I'll move the word 'directly' after 'connected'.